### ISSUE TITLE
Build from Source on openSUSE Tumbleweed

### ISSUE THREAD

**Issue by Yupy** [rethinkdb/rethinkdb#6755]
I have a issue while trying to build rethinkdb from source on openSUSE Tumbleweed.

It fails on make: `https://pastebin.com/dujpHSwc`

To me looks like has to do with openssl-devel but I do have it installed `libopenssl-devel-1.1.1c-1.1.noarch`, do I missed something else?

**Comment by srh** [comment]
Try the head of the v2.3.x branch instead of 2.3.6.  That incompatibility with SSL got fixed since v2.3.6.

I think here might be some other things you might need to fix the build:

* Try `CXX=clang++` on the configure line.
* Try `--fetch npm`.
* See http://samuelhughes.com/rethinkdb/build_instructions.html for inspiration for other ideas.
* If you get it working, please report what you did.  I'll add the info to that web page.

I'll put openSUSE Tumbleweed on my priority list for making build instructions.

**Comment by srh** [comment]
I've updated the instructions.  You'll need to use v2.3.x branch, or the tag v2.3.7 (which currently are the same thing).

```
# Starting from openSUSE server edition
sudo zypper install git-core clang8 gcc libopenssl-devel make \
    patch zlib-devel libcurl-devel
./configure --allow-fetch CXX=clang++
make -j8
```

Maybe `--dynamic jemalloc` is okay, I don't know.
